content/docs/command-reference/exp/init.md

```diff
@@ -17,37 +18,52 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 ## Description
 
 `dvc exp init` helps you quickly get started with experiments. It reduces
-boilerplate for initializing [pipeline](/doc/command-reference/dag) stages in a
-`dvc.yaml` file by assuming defaults about the location of your data,
-[parameters](/doc/command-reference/params), source code, models,
-[metrics](/doc/command-reference/metrics) and
-[plots](/doc/command-reference/plots), which can be customized through config.
+boilerplate DVC procedures by creating a `dvc.yaml` file that assumes default
+location of your input data, <abbr>parameters</abbr>, source code, models,
+<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
+can be customized through the options of this command or via config files.
+Standard repository structure:
 
-It also offers guided `--interactive` mode for creating a stage to be
-[`exp run`](/doc/command-reference/exp/run) later. `dvc exp init` supports
-creating different types of stages, eg: `dl` if you are doing deep learning,
-which uses [dvclive](/doc/dvclive) to monitor and checkpoint progress during
-training of machine learning models.
+```
+├── data/
+├── metrics.json
+├── models/
+├── params.yaml  # required
+├── plots/
+└── src/
+```
+
+> Note that `params.yaml` is the only required file (see `dvc params`).
+
+To use this feature, provide a `command` argument or use the `--interactive`
+(`-i`) mode and answer a few prompts (most of them optional). This wraps the
+command that runs your experiment(s) as a <abbr>stage</abbr> that `dvc exp run`
+can execute.
+
+Different types of stages are supported, such as `dl` (deep learning) which uses
+[DVSLive](/doc/dvclive) to monitor [checkpoints] during training of ML models.
 
-This command is intended to be a quick way to start running experiments. To
-create more complex stages and pipeliens, use `dvc stage add`.
+> This command is intended as a quick way to start running experiments. To
+> codify complex data pipelines, see the `dvc.yaml` specification.
+
+[checkpoints]: /doc/user-guide/experiment-management/checkpoints
 
 ### The `command` argument
 
-The `command` argument is optional, if you are using `--interactive` mode. The
```

Comment:
BTW a question I have is whether to keep this section or move `command` under **Options** like we did for `targets` in https://dvc.org/doc/command-reference/repro#options. I personally like the section more but I remember we discussed with (cc @shcheklein) this and using Options was picked so for consistency I'd move this to Options as well.

> From https://github.com/iterative/dvc.org/pull/3015#pullrequestreview-804146896